modules/core/src/main/java/org/apache/ignite/internal/commandline/ShutdownPolicyCommand.java

```diff
@@ -0,0 +1,105 @@
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.commandline;
+
+import java.util.Comparator;
+import java.util.UUID;
+import java.util.logging.Logger;
+import org.apache.ignite.ShutdownPolicy;
+import org.apache.ignite.internal.client.GridClient;
+import org.apache.ignite.internal.client.GridClientConfiguration;
+import org.apache.ignite.internal.client.GridClientNode;
+import org.apache.ignite.internal.commandline.shutdown.ShutdownPolicyArgument;
+import org.apache.ignite.internal.visor.shutdown.VisorShutdownPolicyTask;
+import org.apache.ignite.internal.visor.shutdown.VisorShutdownPolicyTaskArg;
+import org.apache.ignite.internal.visor.shutdown.VisorShutdownPolicyTaskResult;
+
+import static org.apache.ignite.internal.commandline.CommandList.SHUTDOWN_POLICY;
+
+/**
+ * Command for change or display policy for shutdown.
+ */
+public class ShutdownPolicyCommand implements Command<ShutdownPolicyArgument> {
+    /** Arguments. */
+    private ShutdownPolicyArgument shutdownPolicyArgument;
+
+    /** {@inheritDoc} */
+    @Override public Object execute(GridClientConfiguration clientCfg, Logger logger) throws Exception {
+        try (GridClient client = Command.startClient(clientCfg)) {
+            UUID coordinatorId = client.compute()
+                //Only non client node can be coordinator.
+                .nodes(node -> !node.isClient())
+                .stream()
+                .min(Comparator.comparingLong(GridClientNode::order))
+                .map(GridClientNode::nodeId)
+                .orElse(null);
+
+            VisorShutdownPolicyTaskResult res = TaskExecutor.executeTaskByNameOnNode(
+                client,
+                VisorShutdownPolicyTask.class.getName(),
+                toTaskArg(shutdownPolicyArgument),
+                coordinatorId,
+                clientCfg
+            );
+
+            logger.info("Cluster shutdown policy is " + res.getShutdown());
+        }
+
+        return null;
+    }
+
+    /**
+     * Convert command argument to task argument.
+     *
+     * @param arg Shutdown policy command argument.
+     * @return Argument for shutdown policy task.
+     */
+    private static VisorShutdownPolicyTaskArg toTaskArg(ShutdownPolicyArgument arg) {
+        return new VisorShutdownPolicyTaskArg(arg.getShutdown());
+    }
+
+    /** {@inheritDoc} */
+    @Override public ShutdownPolicyArgument arg() {
+        return shutdownPolicyArgument;
+    }
+
+    /** {@inheritDoc} */
+    @Override public void printUsage(Logger logger) {
+        Command.usage(logger, "Set or display shutdown policy:", SHUTDOWN_POLICY,
+            CommandLogger.optional(CommandLogger.join("|", ShutdownPolicy.values())));
+    }
+
+    /** {@inheritDoc} */
+    @Override public String name() {
+        return SHUTDOWN_POLICY.toCommandName();
+    }
+
+    /** {@inheritDoc} */
+    @Override public void parseArguments(CommandArgIterator argIter) {
```

Comment:
Probably we should add some command parsing test for incorrect shutdown policy somewhere nearby CommandHandlerParsingTest.